utilities/Microsoft.Quantum.Katas/KataMagic.cs

```diff
@@ -169,10 +169,11 @@ public virtual bool Simulate(OperationInfo test, OperationInfo userAnswer, IChan
 
             try
             {
-                var qsim = CreateSimulator(channel);
+                var qsim = CreateSimulator();
 
                 qsim.DisableExceptionPrinting();
                 qsim.DisableLogToConsole();
+                qsim.OnLog += channel.Stdout;
```

Comment:
We may need the same logic to forward histograms here as well.

Reply:
Has been resolved as a repeated attempt with [another set of 3 PRs](https://github.com/microsoft/QuantumKatas/pull/727).